app/services/validator_check_active_service.rb

```diff
@@ -40,19 +40,17 @@ def should_be_destroyed?(validator)
     false
   end
 
-  # number of previous by network
-  def previous_epoch(network)
-    @previous_epochs ||= {}
-    return @previous_epochs[network] if @previous_epochs[network].present?
+  def current_epoch(network)
+    @current_epochs ||= {}
+    return @current_epochs[network] if @current_epochs[network].present?
 
-    current_epoch = EpochWallClock.where(network: network).order(created_at: :desc).last
-    @previous_epochs[network] = current_epoch.epoch - 1
+    @current_epochs[network] = EpochWallClock.where(network: network).order(created_at: :desc).first
```

Comment:
Is there a reason why not just `EpochWallClock.where(network: network).last`? Not a big deal, cause  both queries are quick in this case, but generally the second one is several times quicker.

Reply:
this is related to script/one_time_scripts/back_fill_epoch_data.rb - it's just easier for local development